.github/workflows/build.yml

```diff
@@ -245,6 +245,52 @@ jobs:
           path: |
             build/coreboot.rom
           retention-days: 30
+  test_q35:
```

Comment:
I wonder if this should be in the build.yml. We store the arficatcts from the build workflow, so we should be able to pass them to another test workflow.